web/gui-v2/src/components/ListViewTable.jsx

```diff
@@ -16,12 +18,17 @@ import {
   classes,
 } from '@eto/eto-ui-components';
 
+import AddRemoveColumnDialog from './AddRemoveColumnDialog';
 import HeaderDropdown from './HeaderDropdown';
 import HeaderSlider from './HeaderSlider';
+import GroupSelector from './ListViewGroupSelector';
+import groupsList from '../static_data/groups';
 import columnDefinitions from '../static_data/table_columns';
-import { useMultiState, useWindowSize } from '../util';
-import AddRemoveColumnDialog from './AddRemoveColumnDialog';
-import { Link } from 'gatsby';
+import {
+  commas,
```

Comment:
This is imported but not used I think. Would be nice to have comma formatted numbers in the footer row at least.